### PR TITLE
Fix links to Symfony.com

### PR DESCRIPTION
**Pull request by pich**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,14 +22,14 @@ applications with Symfony. Features include:
 Documentation
 -------------
 
-[Read the Documentation](http://symfony.com/doc/master/bundles/FOSRestBundle/index.html)
+[Read the Documentation](http://symfony.com/doc/current/bundles/FOSRestBundle/index.html)
```
Comment (axzx) [inline README.md:25]:
```suggestion
[Read the Documentation](https://symfony.com/doc/current/bundles/FOSRestBundle/index.html)
```

File: README.md [hunk 2]
```diff
@@ -22,14 +22,14 @@ applications with Symfony. Features include:
...
 
 Please see the upgrade files (`UPGRADING-X.X.md`) for any relevant instructions when upgrading to a newer version.
 
 Installation
 ------------
 
-All the installation instructions are located in the [documentation](http://symfony.com/doc/master/bundles/FOSRestBundle/1-setting_up_the_bundle.html).
+All the installation instructions are located in the [documentation](http://symfony.com/doc/current/bundles/FOSRestBundle/1-setting_up_the_bundle.html).
```
Comment (axzx) [inline README.md]:
```suggestion
All the installation instructions are located in the [documentation](https://symfony.com/doc/current/bundles/FOSRestBundle/1-setting_up_the_bundle.html).
```

**Review by axzx — COMMENTED** [review]
I think you can give a link to the HTTPS version right away

**Comment by xabbuh** [comment]
Thank you @pich.

**Pull request merged**